src/components/Activities.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import { Dropdown, Grid, Button } from "semantic-ui-react";
+import { addActivity } from "../modules/destination.js"
+
+const Activities = props => {
+  const [activity, setActivity] = useState(null);
+  const [actTimes, setActTimes] = useState(null);
+
+  const activities = [
+    { key: 1, value: "amusement park", text: "Amusement Park" },
+    { key: 2, value: "aquarium", text: "Aquarium" },
+    { key: 3, value: "art gallery", text: "Art Gallery" },
+    { key: 4, value: "beauty salon", text: "Beauty Salon" },
+    { key: 5, value: "casino", text: "Casino" },
+    { key: 6, value: "shopping", text: "Shopping" },
+    { key: 7, value: "museum", text: "Museum" },
+    { key: 8, value: "night club", text: "Night Club" },
+    { key: 9, value: "park", text: "Park" },
+    { key: 10, value: "zoo", text: "Zoo" }
+  ];
```

Comment:
have `value` be exactly as Google Places API, e.g. `amusement_park` instead of `amusement park`

Reply:
sure no probs

Reply:
done